src/compressed_tensors/utils/offload.py

```diff
@@ -106,7 +106,19 @@ def update_parameter_data(
         raise ValueError("Attempted to update uninitialized parameter")
 
     dtype = parameter.dtype
-    parameter.data = new_param_data.to(device).to(dtype)
+    try:
+        parameter.data = new_param_data.to(device).to(dtype)
+    except RuntimeError:
```

Comment:
This is better handled by https://github.com/neuralmagic/compressed-tensors/pull/193